.github/workflows/actions/deploy/action.yml

```diff
@@ -45,47 +45,47 @@ runs:
       - name: Get Short SHA
         id: sha
         shell: bash
-        run: echo ::set-output name=short::$(echo "${{ inputs.sha }}" | cut -c -7)
+        run: echo "short=$(echo "${{ inputs.sha }}" | cut -c -7)" >> $GITHUB_OUTPUT
 
       - name: Setup Environment Variables
         id: variables
         shell: bash
         run: |
-             echo ::set-output name=RUN_TEST::"true"
+             echo RUN_TEST="true" >> $GITHUB_OUTPUT
              if [ "${{inputs.environment }}" == "Review" ]
              then
-                 echo ::set-output name=control::$(echo "review" )
+                 echo "control=$(echo "review" )" >> $GITHUB_OUTPUT
```

Comment:
We don't actually need `echo` here and in the subsequent lines. Can we simplify?